.cookiecutterrc

```diff
@@ -63,7 +63,7 @@ default_context:
     test_runner:               'pytest'
     travis:                    'yes'
     travis_osx:                'yes'
-    version:                   '0.4.0.dev0'
+    version:                   '0.4.0.dev1'
```

Comment:
Should this be v0.5.1?

Reply:
I think this file was used to create the project and is no longer needed.